src/transactions.rs

```diff
@@ -737,13 +737,8 @@ impl UnvaultTransaction {
     /// # Panic
     /// Will panic if passed a csv higher than
```

Comment:
You aren't passing a csv anymore though?

Reply:
Ah my bad, now I get it (I was reviewing commit by commit)

Reply:
Oh, no you are right, it should be part of the precedent commit. Will leave it as is though it's a nit :)